src/index.js

```diff
@@ -251,7 +251,7 @@ class CID {
    * @returns {String}
    */
   [Symbol.for('nodejs.util.inspect.custom')] () {
-    return 'CID(' + this.toString() + ')'
+    return 'CID(' + this.toString(this.multibaseName) + ')'
```

Comment:
but this ends up at `toBaseEncodedString (base = this.multibaseName)` so shouldn't this be in effect anyway?

In a Node REPL:

```
> new CID('QmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n')
CID(QmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n)
```

Looks like it's working to me already.

Reply:
hmm looks like cid is changed during one of module i use